src/core/src/service.rs

```diff
@@ -289,6 +351,126 @@ mod tests {
         };
     }
 
+    #[test]
+    fn test_get_blocks(){
+        let mut test_mock_str = HashMap::new();
+        test_mock_str.insert(String::from("test_min"), String::from("10"));
+        debug!("In Mocking get block");
+        let block_id_map = BlockId::from(vec![16,16,85,32]);
+        let block_id_test = BlockId::from(vec![16,16,85,32]);

```

Comment:
Does having repetitive block ids(16) here serve any purpose?

Reply:
I don't think so as it's a map so key should be unique.